src/Util/FileUploader.php

```diff
@@ -2,96 +2,78 @@
 
 namespace App\Util;
 
-use Gedmo\Sluggable\Util\Urlizer;
-use phpDocumentor\Reflection\DocBlock\Tags\Var_;
+use League\Flysystem\FilesystemInterface;
 use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\HttpFoundation\Request;
 
 class FileUploader
 {
     /**
-     * The directory where uploaded files will be placed.
+     * Directory prefix.
      *
      * @var string
      */
-    private $uploadDirectory;
+    private $prefix;
 
     /**
-     * The directory where file chunks will be stored.
+     * An instance of the Flysystem interface.
      *
-     * @var string
+     * @var FilesystemInterface
      */
-    private $chunksDirectory;
+    private $flysystem;
 
-    public function __construct(string $homedirPrefix)
+    /**
+     * Relative path for chunks folder.
+     */
+    const CHUNKS_DIRECTORY = 'upload/chunks';
+
+    /**
+     * Relative path for files folder.
+     */
+    const FILES_DIRECTORY = 'upload/files';
+
+    /**
+     * FileUploader constructor.
+     *
+     * @param string              $homedirPrefix    Home directory path prefix.
```

Comment:
Not really using these anymore right?

Reply:
But we need some kind of directory prefix